### ISSUE TITLE
ParameterAlignment autocorrect doesn't align params after Layout/MultilineMethodArgumentLineBreaks does its thing

### ISSUE THREAD

**Issue by nc-williamss** [rubocop-hq/rubocop#7689]
I attempted to add a new cop (Layout/MultilineMethodArgumentLineBreaks) to move all params to a multiline arg layout, and the AlignParameters cop didn't do enough to make me able to use this newer cop...
--------

## Expected behavior

```ruby
def get_equity(log)
  variablex = "derp"
  something = "derp2"
  try_path(log, variablex,
    something)
end

# should correct to:
def get_equity(log)
  variablex = "derp"
  something = "derp2"
  try_path(
    log,
    variablex,
    something)
end
```

## Actual behavior

```ruby
def get_equity(log)
  variablex = "derp"
  something = "derp2"
  try_path(log, derp,
    something)
end

# actually corrects to:
def get_equity(log)
  variablex = "derp"
  something = "derp2"
  try_path(
    log,
variablex,
    something)
end
```

## Steps to reproduce the problem
Use the following cops:
```yml
Layout/MultilineMethodArgumentLineBreaks:
  Enabled: true
Layout/ParameterAlignment:
  Enabled: true
  EnforcedStyle: with_first_parameter
Layout/ArrayAlignment:
  Enabled: true
```

(note, i'm using additional cops, but i've deemed these as the relevant one... let me know if there are any other which might need to be specified here)
Also worth noting, the multiline method arg cop doesn't strictly need to be applied to see this issue... 

```ruby
  try_path(
    log,
variablex,
    something)
```
this example won't correct _AT ALL_

Run with `--safe-auto-correct` and the result is the above example...
I would expect the MultilineMethodArgumentLineBreaks to move my param, then I would expect the ParameterAlignment cop to clean up the awkward lack of indentation. 

## RuboCop version
```
rubocop -V
0.79.0 (using Parser 2.7.0.2, running on ruby 2.5.3 x86_64-darwin18)
```

**Comment by brandonjmckay** [comment]
Noticing the same behaviour.
In my project I've copied over the AirBnB's rubocop setup files and updated them for rubocop `0.80` because that's the latest that codeclimate seems to support currently.

`ParameterAlignment` seems to do nothing for me at all.

Even tried running autocorrect on:
```ruby
foo :bar,
                                 :baz,
:cheadle
```
It had no effect on that.

**Comment by laurmurclar** [comment]
Still seeing this in `0.83.0`

**Comment by brandonjmckay** [comment]
activity
